core/src/test/scala/org/apache/spark/sql/test/SharedSQLContext.scala

```diff
@@ -398,7 +402,24 @@ object SharedSQLContext extends Logging {
       import com.pingcap.tispark.TiConfigConst._
 
       pdAddresses = getOrElse(prop, PD_ADDRESSES, "127.0.0.1:2379")
-      dbPrefix = getOrElse(prop, DB_PREFIX, "tidb_")
+      catalogPluginMode =
+        if (TiSparkInfo.SPARK_MAJOR_VERSION == "3.0") getFlagOrFalse(prop, USE_CATALOG_PLUGIN)
+        else false
+      dbPrefix = if (catalogPluginMode) "" else getOrElse(prop, DB_PREFIX, "tidb_")
+
+      // properties for ticatalog plugin
+      if (catalogPluginMode) {
+        if (!prop.containsKey("spark.sql.catalog.tidb_catalog")) {
+          prop
+            .put(
+              "spark.sql.catalog.tidb_catalog",
+              "org.apache.spark.sql.catalyst.catalog.TiCatalog"
+            )
+        }
+        if (!prop.containsKey("spark.sql.catalog.tidb_catalog.pd.address")) {
+          prop.put("spark.sql.catalog.tidb_catalog.pd.address", pdAddresses)
```

Comment:
what if use spark-shell ? we should add the parameter `spark.sql.catalog.tidb_catalog.pd.address` for normal users

Reply:
user should set `spark.sql.catalog.tidb_catalog` and `spark.sql.catalog.tidb_catalog.pd.address` in spark configuration file, if `spark.sql.catalog.tidb_catalog.pd.address` is not set, `127.0.0.1:2379` will be used as the default value